### PR TITLE
[for discussion] mimir lib

### PR DESCRIPTION
**Pull request by pbougue**
For discussion
Not sure about the cargo.toml/lib.rs, the fact that one must know lib mimir externally (see tests), the way its stored/exposed
It compiles and tests in stable.

### CODE REVIEW

File: libs/mimir/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+extern crate log;
+extern crate env_logger;
+
+extern crate serde;
+extern crate serde_json;
+
+pub mod objects;
+pub mod rubber;
+
+pub use objects::{Addr, Street, Admin, Coord};
```
Comment (antoine-de) [inline libs/mimir/src/lib.rs:43]: i'm not a fan of this, but I think there we can import objects::*
since they represent `mimir`

File: build.rs [hunk 2]
```diff
@@ -1,32 +1,3 @@
-// this file is used to be able to use serde (https://github.com/serde-rs/serde) in rust stable
-// cf serde readme for explanations
+// this file is used to be able to use OUT_DIR env variable
```
Comment (antoine-de) [inline build.rs:1]: you're sure you need this ?

**Comment by antoine-de** [comment]
you'll have tons of conflicts with the other running PR :wink:

**Comment by pbougue** [comment]
PR will be done directly on CanalTP repo

**Pull request closed without merging**